test/problems/multipass/output_validators/multipass_validator/multipass_validator.py

```diff
@@ -2,18 +2,36 @@
 import sys
 import pathlib
 
+judgemessage = pathlib.Path(sys.argv[3]) / 'judgemessage.txt'
```

Comment:
'we' kinda always just use `stderr` instead of `judgemessage`, but yes I suppose this is better for a proper example

Reply:
I would say ideally both should have an example? the system should be able to concat `judgemessage.txt` as well as `stderr` output.

Reply:
I did not use `stderr` because I am not sure how it will be displayed in other systems like Kattis. I am fine with adding one message on stderr, if you think it makes sense though

Reply:
For normal problems both work.
Probably it's indeed good to check that both work for multipass as well.
Maybe print to both a message like `this is printed both to stderr and judgerror` or so.